metrics-datadog/src/test/java/org/coursera/metrics/datadog/TagsMergerTest.java

```diff
@@ -0,0 +1,44 @@
+package org.coursera.metrics.datadog;
+
+import org.junit.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.TreeSet;
+
+public class TagsMergerTest {
```

Comment:
Please also add a test to `DatadogReporterTest` ensuring that the callback is invoked when configured